isc_dhcp_leases/iscdhcpleases.py

```diff
@@ -108,9 +111,13 @@ class IscDhcpLeases(object):
         r"ia-(?P<type>ta|na|pd) \"(?P<id>[^\"\\]*(?:\\.[^\"\\]*)*)\" {(?P<config>[\s\S]+?)\n}")
     regex_iaaddr = re.compile(r"ia(addr|prefix) (?P<ip>[0-9a-f:]+(/[0-9]+)?) {(?P<config>[\s\S]+?)\n\s+}")
 
-    def __init__(self, filename, gzip=False):
+    def __init__(self, filename, gzip=False, now=None):
+        if not datetime_is_ok(now):
+            raise ValueError
```

Comment:
Can't you pass a descriptive error message to ValueError? I think the user shouldn't have to dive through the stack trace if things go wrong.

Reply:
Yeah, that's a good point. Thanks for the review, I'll push an update shortly.

Reply:
I converted `datetime_is_ok()` (returning boolean) to `check_datetime()` (directly raising `ValueError` when appropriate).

I also tweaked the test to *really* only accept `None` or timezone-aware datetime objects.